app/controllers/browse_controller.rb

```diff
@@ -40,7 +40,7 @@ def show_html(page)
 
     configure_ab_response if page_in_ab_test?
 
-    if taxon_resolver.taxon_base_path
+    if taxon_resolver.redirect?
```

Comment:
From just reading this, I'm not sure why there is a conditional redirect here. Is there another name for the `redirect?` method that would make the purpose of this code clearer?

Reply:
The `TaxonRedirectResolver#redirect?` method should have a different name? I'm not sure what it would be. To be fair, there is a lot of conditional logic in these controllers. It's not immediately obvious to me how to further simplify them though...

Reply:
(We discussed this in person, and still didn't come out with a new name, so maybe it doesn't exist)